src/web/client/WebExtensionContext.ts

```diff
@@ -627,6 +661,99 @@ class WebExtensionContext implements IWebExtensionContext {
     public getVscodeWorkspaceState(key: string): IEntityInfo | undefined {
         return this._vscodeWorkspaceState.get(key);
     }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    public async getWorkerScript(workerUrl : URL) : Promise<any> {
+        try {
+            this.telemetry.sendInfoTelemetry(
+                telemetryEventNames.WEB_EXTENSION_FETCH_WORKER_SCRIPT,
+                { workerUrl: workerUrl.toString() }
```

Comment:
I don't we need to send workerUrl to telemetry

Reply:
done!